spec/ddtrace/contrib/grape/tracer_spec.rb

```diff
@@ -196,6 +196,34 @@
     end
 
     context 'failure' do
+      context 'without filters' do
+        subject(:response) { post '/base/hard_failure' }
+
```

Comment:
I would explicitly add ` let(:configuration_options) { { dont_report_4xx: false } }` here so it's clear we're testing that errors get set without `dont_report_4xx`